### PR TITLE
Channel.py

### PR DESCRIPTION
**Pull request by Vaishnavofficial**
Creates the channel with command

### CODE REVIEW

File: plugins/create.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    'description': "Creates a channel with a name"},
+    allow_channels=False, allow_bots=False)
+async def create_ch(message:Message):
+    try:
+      args = message.text.split(None, 1)
+      title = args[1]
+      await userge.create_channel(title, 'nice')
+      await message.edit(f"`successfully made a new channel {title}`")
+    except Exception as e:
+       message.edit("something went wrong")
```
Comment (rking32) [inline plugins/create.py:18]: missing `await` here

**Comment by rking32** [comment]
also, can you try to fix [these](https://github.com/UsergeTeam/Userge-Plugins/pull/242#issuecomment-1022987477) little things?

**Comment by Jigarvarma2005** [comment]
add a flag or syntax for custom description

#suggestion

**Comment by Krishna-Singhal** [comment]
> add a flag or syntax for custom description (must check description character limit)
> 
> #suggestion

How much limit ?

**Comment by Jigarvarma2005** [comment]
.strip() will not convert "This Text" to "ThisText" ???

**Pull request merged**